docs/preview/duckdb/backups_and_recovery.md

```diff
@@ -0,0 +1,111 @@
+
+- [SQL dump](https://www.postgresql.org/docs/current/backup-dump.html): This approach is similar to the one mentioned for SQLite and DuckDB. This process can happen periodically and can only recover to a particular point in time (i.e., the time of the dump). For DuckLake, this will be a specific snapshot, and transactions after this snapshot will not be recorded.
+- [Continuous Archiving and Point-in-Time Recovery (PITR)](https://www.postgresql.org/docs/current/continuous-archiving.html): This is a more complex approach but allows recovery to a specific point in time. For DuckLake, this means you can recover to a specific snapshot without losing any transactions.
+
+Nothe that the SQL dump approach can also be managed by DuckDB using the [Postgres extension](https://duckdb.org/docs/stable/core_extensions/postgres). In fact, the backup can be a DuckDB file.
+
+```sql
+-- Backup
+ATTACH 'postgres:connection_string' AS main (READ_ONLY);
+ATTACH 'duckdb:backup.db' AS backup;
+COPY DATABASE FROM main TO backup;
+
+-- Recover
+ATTACH 'postgres:connection_string' AS main;
+ATTACH 'duckdb:backup.db' AS backup (READ_ONLY);
+COPY DATABASE FROM backup TO main;
+ATTACH 'ducklake:postgres:connection_string' AS my_ducklake;
+```
+
+> Cloud-hosted PostgreSQL solutions may offer different mechanisms. We encourage you to check what your specific vendor recommends as a strategy for backup and recovery.
+
+## Storage Backup and Recovery
+
+Backup and recovery of the data files also depend on the storage you are using. In this document, we will only focus on cloud-based object storage since it is the most common for Lakehouse architectures.
+
+### S3
+
+In S3, there are three main mechanisms that AWS offers to back up and/or restore data:
+
+- [Cross-bucket replication](https://docs.aws.amazon.com/AmazonS3/latest/userguide/replication.html)
+- [S3 backup service](https://docs.aws.amazon.com/aws-backup/latest/devguide/s3-backups.html)
+- [Enable S3 versioning](https://docs.aws.amazon.com/AmazonS3/latest/userguide/Versioning.html)
+
+Both the S3 backup service and S3 object versioning will restore data files in the same bucket. On the other hand, cross-bucket replication will copy the files to a different bucket, and therefore your DuckLake initialization should change:
+
+```sql
+-- Before
+ATTACH 'ducklake:some.db' AS my_ducklake (DATA_PATH 's3://og-bucket/')
+
+-- After
+ATTACH 'ducklake:some.db' AS my_ducklake (DATA_PATH 's3://replication-bucket/')
+```
+
+### GCS
+
+GCS has similar mechanisms to back up and/or restore data:
+
+- [Cross-bucket replication](https://cloud.google.com/storage/docs/using-cross-bucket-replication)
+- [Backup and DR service](https://cloud.google.com/backup-disaster-recovery/docs/concepts/backup-dr)
+- [Object versioning](https://cloud.google.com/storage/docs/object-versioning) with soft deletes enabled
+
+Regarding cross-bucket replication, repointing to the new bucket will be necessary.
```

Comment:
Can this be done once?
I think there might be a way to store the DATA_PATH (and allow to then use default one)